app/admin/quests/dashboard/[questId]/page.tsx

```diff
@@ -282,7 +282,6 @@ export default function Page({ params }: QuestIdProps) {
 
     const updatedTasks = filteredSteps.filter((step, index) => {
       return (
```

Comment:
The filteredSteps already eliminates steps with the type of "None" hence the error